CRM/Atomfeeds/Page/Feed.php

```diff
@@ -78,20 +78,9 @@ private function _getBlog() {
     // Fetch data from cache
     // TODO: Once we support multiple feeds, we need to store the feed
     // ID or something like that in the cache.
-    $cache = CRM_Core_DAO::executeQuery("SELECT data, created_date FROM civicrm_cache
-      WHERE group_name = 'dashboard' AND path = 'feed'");
-    if ($cache->fetch()) {
-      $expire = time() - (60 * 60 * 24 * self::CACHE_DAYS);
-      // Refresh data after CACHE_DAYS
-      if (strtotime($cache->created_date) < $expire) {
-        $new_data = $this->_getFeed($this->getBlogUrl());
-        // If fetching the new feed was successful, return it
-        // Otherwise use the old cached data - it's better than nothing
-        if ($new_data) {
-          return $new_data;
-        }
-      }
-      return unserialize($cache->data);
+    $blogData = Civi::cache('dashboard')->get('feed');
```

Comment:
note the above code should be covered in the Civi::cache stuff

Reply:
It's been a while since I've been working with CiviCRM, but I really like this new caching interface.